### ISSUE TITLE
Related work

### ISSUE THREAD

**Issue by gh540-svg** [apple/ml-ssd#9]
Great work! We recently developed a related method that also does not rely on ground truth answers or external verifiers. Our approach, Self-Policy Distillation (SPD), steers self-generated training data by projecting KV activations into a capability-selective subspace, and then fine-tunes the model on the resulting outputs using standard next-token prediction loss. 

X :
https://x.com/zhuokaiz/status/2058265358134337549
arXiv:
https://arxiv.org/pdf/2605.22675

If you find our research relevant, we would sincerely appreciate your consideration of citing it in your work.
